admin/class-ajax.php

```
@@ -223,7 +223,7 @@ public function details() {
 		if ( $rules ) {
 
 			// if ANWW is active remove link_blank for details meta box.
-			if ( EDAC_ANWW_ACTIVE ) {
+			if ( ANWW_VERSION ) {
```

PHPStan (2.0.3)

Constant ANWW_VERSION not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)